.travis/analyse.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 function finish() {
-  make logs stop
+  make logs && make stop
```

Comment:
`make logs; make stop`?

Reply:
Yes, safer to always stop even in case of `docker-compose logs` failing.